wkcuber/cubing.py

```diff
@@ -234,6 +246,13 @@ def cubing_job(
                 raise exc
 
 
+def get_channel_count_and_dtype(source_path: str) -> Tuple[int, str]:
+    source_files = find_source_filenames(source_path)
```

Comment:
I added your suggestion for future users, but currently it's not needed because the image stack converter never gets called when no source files are found.